### PR TITLE
Update to the Einstein apps import URI

### LINKED ISSUE #2931: Update docs to the latest Einstein release and other associated configuration

### PR DESCRIPTION
**Pull request by ilayaperumalg**
- update the latest versions for the apps

Resolves #2931

### CODE REVIEW

File: spring-cloud-dataflow-docs/src/main/asciidoc/index.adoc [hunk 1]
```diff
@@ -24,13 +24,13 @@ Sabby Anandan; Marius Bogoevici; Eric Bottard; Mark Fisher; Ilayaperumal Gopinat
...
-:docker-http-source-kafka-version: 1.3.1.RELEASE
-:docker-time-source-kafka-version: 1.3.1.RELEASE
-:docker-timestamp-task-version: 1.3.1.RELEASE
+:docker-http-source-rabbit-version: 2.1.0.RELEASE
+:docker-time-source-rabbit-version: 2.1.0.RELEASE
+:docker-log-sink-rabbit-version: 2.1.0.RELEASE
+:docker-log-sink-kafka-version: 2.1.0.RELEASE
+:docker-http-source-kafka-version: 2.1.0.RELEASE
+:docker-time-source-kafka-version: 2.1.0.RELEASE
+:docker-timestamp-task-version: 2.1.0.RELEASE
```
Comment (sabbyanandan) [inline spring-cloud-dataflow-docs/src/main/asciidoc/index.adoc]: We can't switch `timestamp` to 2.1 GA yet; we are waiting on SCDF 2.0 to release the Task train, so it'd be good to revert this to 1.3.1 or 2.0.1.

File: spring-cloud-dataflow-docs/src/main/asciidoc/streams.adoc [hunk 2]
```diff
@@ -23,7 +23,7 @@ The general lifecycle of a Stream is:
 . Create a Stream Definition.
 . Deploy the Stream.
 . Undeploy or Destroy the Stream.
-. Upgrade or Rollack applications in the Stream.
+. Upgrade or Rollback applications in the Stream.
```
Comment (sabbyanandan) [inline spring-cloud-dataflow-docs/src/main/asciidoc/streams.adoc]: 👏

**Comment by chrisjs** [comment]
no additional comments for me besides what @sabbyanandan mentioned

**Comment by ilayaperumalg** [comment]
Addressed the review comments to update the latest on docker compose config and task apps release URI.

File: spring-cloud-dataflow-docs/src/main/asciidoc/index.adoc [hunk 3]
```diff
@@ -24,13 +24,13 @@ Sabby Anandan; Marius Bogoevici; Eric Bottard; Mark Fisher; Ilayaperumal Gopinat
...
-:docker-http-source-kafka-version: 1.3.1.RELEASE
-:docker-time-source-kafka-version: 1.3.1.RELEASE
-:docker-timestamp-task-version: 1.3.1.RELEASE
+:docker-http-source-rabbit-version: 2.1.0.RELEASE
+:docker-time-source-rabbit-version: 2.1.0.RELEASE
+:docker-log-sink-rabbit-version: 2.1.0.RELEASE
+:docker-log-sink-kafka-version: 2.1.0.RELEASE
+:docker-http-source-kafka-version: 2.1.0.RELEASE
+:docker-time-source-kafka-version: 2.1.0.RELEASE
+:docker-timestamp-task-version: 2.0.1.RELEASE
```
Comment (sabbyanandan) [inline spring-cloud-dataflow-docs/src/main/asciidoc/index.adoc]: Sorry for the wrong info in the previous review. We don't have 2.0.1 yet, the latest is still at 2.0.0 for `timestamp`.

Comment (chrisjs) [reply spring-cloud-dataflow-docs/src/main/asciidoc/index.adoc]: done on merge

**Pull request closed without merging**